pkg/telemetry/stats_telemetry.go

```diff
@@ -0,0 +1,41 @@
+package telemetry
+
+import "sync"
+
+// StatsTelemetryHandler contains methods needed for sending stats metrics
```

Comment:
This is a subset of an the aggregator's interface, right?  Perhaps a comment about that?